meetings/2021-11-08.md

```diff
@@ -0,0 +1,158 @@
+* AC: Yes
+* AC: Agreed, such scopes may belong more to the AuthZ spec than to the AuthN spec. However, there is a lot of overlap between specs there. Should Solid-OIDC mention this at least?
+* eP: we should only need to say that WWW-Authenticate is the mechanism to discover how an entity can get access to a resource.
+* AC: This would make the specification stronger, and simpler, with the caveat that we should be careful about how to redirect implementers to additional AuthZ specifications without being too overwhelming to discover how to approach these scopes.
+* eP: one issue is we then depend on the authorization spec to define that. The ecosystem would need the authorization spec as a MUST.
+* AC: What would be the next specs for the Solid-OIDC spec ?
+* eP: next steps would be to introduce the authorization server. E.g., www-authenticate `as_uri`
+* ...: remove access token
+* ...: bind DPoP token to ID token
+* ...: describe claims pushing, `id_token` used as claim
+* AC: We'll want to be communicate clearly to the wider Solid community what these changes entail, and what path should be followed for transition.
+* eP: I will invite a CSS representative to make sure they are aware of this. Defining this as_uri should be fairly simple. Token introspection should be able to occur internally
+* ...: it would be good to describe these changes in a pull request to Solid-OIDC.
+* NAS: we will also need to update the client libraries. The client-authn libraries do not support UMA. It also assumes that the RS supports the access tokens from an IdP.
+* ...: We will want to make sure there is client support before asking servers to change
+* eP: as a maintainer of a client library, how do you see this transition period? Do you need clear documentation of these changes?
+* NAS: it is a bit early to asses the impact. It isn't too problematic to have the two live side-by-side. An RS can advertise this capability. The transition period is more for the client libraries to adapt
+* eP: if both the client and server have a transition period, then there is a path for both to move forward
+* ...: there was discussion from a few weeks ago about keeping access tokens in a closure, so it might also be possible to store the ID token like that
+* NAS: there are some things to double-check there. For refresh, we rely on the storage of an ID token stored in local storage.
+* eP: there is an open PR that starts this (??? add URL). What would be the best workflow
+* AC: one first step might be to describe the DPoP-bound ID token
+* eP: with access tokens, we have a notion of refreshing. how will that work with ID tokens?
+* NAS: what would be the expected lifetime of an ID token? Do we expect that the lifespan would be longer lived?
+* AC: It should be short-lived too, because it you can trade the ID token for an access token, then there is no difference between having a long-lived AT and a long-lived ID Token
```

Comment:
```suggestion
* AC: It should be short-lived too, because if you can trade the ID token for an access token, then there is no difference between having a long-lived AT and a long-lived ID Token
```